lua/entities/gmod_tardis_interior/cl_init.lua

```diff
@@ -7,4 +7,8 @@ ENT:AddHook("PlayerInitialize", "interior", function(self)
     self.Fallback=self.metadata.Interior.Fallback
     self.Portal=self.metadata.Interior.Portal
     self.ExitDistance=self.metadata.Interior.ExitDistance
-end)
\ No newline at end of file
+    if GetConVarNumber("r_rootlod")>0 then
```

Comment:
Can we move this out to a separate module? Something like `TARDIS/lua/entities/gmod_tardis_interior/modules/cl_lod_prompt.lua`

Also hooking on `PlayerEnter` may make a bit more sense so that we're only bothering people interacting with the TARDIS in a multiplayer scenario rather than everyone.

Can we add a check as well to see if there is an optional thing set on the metadata, maybe `self.metadata.Interior.RequireHighModelDetail`? If it is not set (nil) then assume that it is required. This will allow extension developers to disable the prompt if not required for their interior.